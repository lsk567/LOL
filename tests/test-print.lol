int a = 5;
println(str_of_int(a));
println(str_of_int(5));

float b = 1.234567;
println(str_of_float(b));
println(str_of_float(1.234567));
