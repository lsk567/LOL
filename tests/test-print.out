5
5
1.23457
1.23457
